gerrit-entrypoint.sh

```diff
@@ -83,13 +83,18 @@ EOF
 done
 
   #Customize gerrit.config
+  [ -z "${MAKE_FIRST_USER_ADMIN}" ] || set_gerrit_config capability.makeFirstUserAdmin "${MAKE_FIRST_USER_ADMIN}"
+  [ -z "${ENABLE_SIGNED_PUSH}" ] || set_gerrit_config receive.enableSignedPush "${ENABLE_SIGNED_PUSH}"
+  [ -z "${DOWNLOAD_SCHEME}" ] || set_gerrit_config download.scheme "${DOWNLOAD_SCHEME}"
```

Comment:
The download.scheme options can be set multi-times. This will make it only support one scheme.